options_write.go

```diff
@@ -35,6 +35,22 @@ func (opts *WriteOptions) DisableWAL(value bool) {
 	C.rocksdb_writeoptions_disable_WAL(opts.c, C.int(btoi(value)))
 }
 
+// SetLowPri if true, this write request is of lower priority if compaction is
+// behind. In this case, no_slowdown = true, the request will be cancelled
+// immediately with Status::Incomplete() returned. Otherwise, it will be
+// slowed down. The slowdown value is determined by RocksDB to guarantee
+// it introduces minimum impacts to high priority writes.
+//
+// Default: false
+func (opts *WriteOptions) SetLowPri(value bool) {
+	C.rocksdb_writeoptions_set_low_pri(opts.c, boolToChar(value))
+}
+
+// IsLowPri returns if the write request is of lower priority if compaction is behind.
+func (opts *WriteOptions) IsLowPri() bool {
+	return charToBool(C.rocksdb_writeoptions_get_low_pri(opts.c))
+}
```

Comment:
Should we call it GetLowPri to match?

Reply:
I had just copied this out of grocksdb... Given that charToBool didn't exist, I don't think there was any `Is` type check anywhere else?